src/Features/CSharp/Portable/CodeFixes/GenerateVariable/GenerateVariableCodeFixProvider.cs

```diff
@@ -24,11 +24,10 @@ internal class GenerateVariableCodeFixProvider : AbstractGenerateMemberCodeFixPr
         private const string CS0117 = nameof(CS0117); // error CS0117: 'TestNs.Program' does not contain a definition for 'blah'
         private const string CS0539 = nameof(CS0539); // error CS0539: 'Class.SomeProp' in explicit interface declaration is not a member of interface
         private const string CS0246 = nameof(CS0246); // error CS0246: The type or namespace name 'Version' could not be found
+        private const string CS0120 = nameof(CS0120); // error CS0120: An object reference is required for the non-static field, method, or property 'A'
```

Comment:
This just means the fixer *runs*.  It still does checking to make sure it is appropriate to add the fix.

This is yet another one of those cases where we have to exhaustively know every single error the compiler might emit when the thing you want is missing.  It's *super* annoying, and i really wish we didn't use this mechanism for code fix providers.